### PR TITLE
Jj improve tsm import pattern

### PR DESCRIPTION
**Pull request by JorMaister**

### CODE REVIEW

**Review by pconesa — APPROVED** [review]
Nice, and with a test!!

File: tomo/convert/mdoc.py [hunk 1]
```diff
@@ -384,7 +384,8 @@ def _validateMdocInfoRead(self, ignoreFilesValidation=False):
                        'ExposureDose or FrameDosesAndNumber or '
                        '(DoseRate and ExposureTime) or '
                        '(MinMaxMean and CountsPerElectron)\n')
-        if not self.getTiltAxisAngle():
+        tAx = self.getTiltAxisAngle()
+        if tAx is not None and tAx != '':  # 0.0 value must be allowed
```
Comment (azazellochg) [inline tomo/convert/mdoc.py:388]: this will raise validation error every time!

**Comment by azazellochg** [comment]
how was this reviewed/tested when it broke import mdoc validation?

**Comment by azazellochg** [comment]
I have reverted this change: https://github.com/scipion-em/scipion-em-tomo/commit/1df24b73c0c8b13bc386944ee7718ec0a12eca76 in devel until it is properly implemented and reviewed in another PR

**Comment by pconesa** [comment]
I should have run the tests. Overconfidence is a mistake!

**Pull request merged**